roboteam_ai/src/control/controlGoToPos/ControlGoToPosLuTh.cpp

```diff
@@ -111,12 +111,11 @@ ControlGoToPosLuTh::Command ControlGoToPosLuTh::goToPos(RobotPtr robot, Vector2
 
         if (! pidInit) {
             pidInit = true;
-            pid.initialize(1.0/rtt::ai::constants::tickRate);
-            pid.setParams(3.0, 0.0, 0.2, 0.0, 0.0, 0.0);
+            pid.setPID(3.0, 0, 0.5);
```

Comment:
Why these values?

Reply:
yes. - pid is gewoon magic numbers. niks aan te doen

Reply:
They still can be structured, i.e. by putting them on top of the file as constants or in the header file.